src/OmniSharp.Abstractions/OmniSharpEndpoints.cs

```diff
@@ -65,6 +65,8 @@ public static class V2
             public const string CodeStructure = "/v2/codestructure";
 
             public const string Highlight = "/v2/highlight";
+
+            public const string QuickInfo = "/v2/quickinfo";
```

Comment:
I think it doesn't need to be v2 since there never was a quickinfo v1

Reply:
I considered the existing TypeInfo v1. I can change it though.